rabbitmq-core/src/FmuContainerCore.cpp

```diff
@@ -356,6 +380,24 @@ void showValue(ostream &os, const char *prefix, date::sys_time<std::chrono::mill
        << " Value: " << val.second << "\n";
 }
 
+void FmuContainerCore::sendCheckCompose(pair<string,string>input, string &message){
+    if(!message.empty()){
```

Comment:
Same string is used both places.

Reply:
Yes. If the string is empty, it will be initialised in the `else` block. Otherwise, the next input will be appended to a non string in the `if` block.